colpali_engine/collators/corpus_query_collator.py

```diff
@@ -0,0 +1,78 @@
+from random import randint
+from typing import Any, Dict, List, Optional, cast
+
+from datasets import Dataset
+
+from colpali_engine.collators.visual_retriever_collator import VisualRetrieverCollator
+from colpali_engine.utils.processing_utils import BaseVisualRetrieverProcessor
+
+
+class CorpusQueryCollator(VisualRetrieverCollator):
+    def __init__(
+        self,
+        processor: BaseVisualRetrieverProcessor,
+        max_length: int = 2048,
+        image_dataset: Optional[Dataset] = None,
+        mined_negatives: bool = True,
```

Comment:
Could you rename `mined_negatives` into `use_mined_negatives`? I think it's better since it makes it clear it is a boolean.